news/_posts/2020-03-11-improvements-on-contributors-board.md

```diff
@@ -0,0 +1,75 @@
+
+There are many ideas that are currently being tested, or will be soon:
+
+- [Call for papers at PrestaShop events](https://build.prestashop.com/news/CFP-PrestaShop-Day-Paris-2020/), giving a place for community speakers
+- Opening the maintainer role on a GitHub repository with selected developers, in order to identify problems and document how to do it well;
+- Opening a chat platform for contributors (due date on Q1)
+- ... and this is just the tip of the iceberg
+
+But, at the end, the question remains: how can a contributor show their knowledge, their expertise, and their investment in the project to their customers.
+
+This is why it is time to improve the Contributors' board. Let's see what is on the schedule already.
+
+
+## The current plan
+
+### First step: quick wins
+
+If you go on [contributors.prestashop.com](http://contributors.prestashop.com/), it is easy to notice a few issues:
+
+- Some contributors like PrestaShop employees are not listed, although they do a huge job on the open source project since the very beginning, and this work is currently hidden
+- It is not possible to see the entire list of contributors, without clicking on "see more", or on page numbers
+- The board lists only contributions to the core; but the PrestaShop software is also made of modules, and of course, there are other very important contributions like specifications, documentation, blog articles, and many more. 
+
+So, it became obvious that the first step must be to show the work of everyone involved in the making of the PrestaShop open source software project.
+
+In the weeks to come, a few improvements will be in production:
+
+1. Remove the current filter to hide some contributors, in order to see everybody. Like the current version, the number of commits will be used, that part doesn't change
+2. Instead of just counting commits on the core repository, it is time to count commits on all the repositories; this is easier now that the PrestaShop GitHub's organisation is focused only on the open source project
+3. The page will be modified a bit to be fully scrollable, without hidden parts, with the aim to make it easy to find someone 
+
+These will be the first improvements, and the development work has been started already. The result is subject to evolve a bit at the beginning, depending on how data will be aggregated, and also because the cleanup of the GitHub organization is not yet complete (e.g: there are still some old partner modules that need to be archived). However, the new board should quickly give a comprehensive view of the entire contributor team over years.
```

Comment:
```suggestion
These will be the first improvements, and the development work has been started already. The result is subject to evolve a bit at the beginning, depending on how data will be aggregated, and also because the cleanup of the GitHub organization is not yet complete (there are still some old partner modules that need to be archived). However, the new board should quickly give a comprehensive view of the entire contributor team over the years.
```